libs/elements/src/lib/form-validation/vl-form-validation.lib.js

```diff
@@ -6999,15 +7006,17 @@
      * @param       {[object]}       config [the config file to validate against]
      */
 
+    // feat: UIG-2550 - vl-form - validatie voor input fields met een dot (.) in het name attribuut
     var _validateField = function _validateField(el, config) {
+        const name = config.escapeFieldNames ? el.name.replace(/\./g, '\\\\\\\\\\.') : el.name;
```

Comment:
zou die andere omstandigheden mss maar 1 keer definiëren en hergebruiken maar gezien we in een file werken +7000 lijnen willen we ook niet teveel heen en weer klikken door dingen naar buiten te werken